src/Send/SendScreen/SendScreen.tsx

```diff
@@ -156,14 +156,14 @@ export const SendScreen = () => {
   }
 
   return (
-    <SafeAreaView edges={['left', 'right']} style={styles.container}>
+    <SafeAreaView edges={['left', 'right', 'bottom']} style={styles.container}>
       <StatusBar type="dark" />
 
       <ErrorBanners />
 
       <AvailableAmountBanner />
 
-      <ScrollView style={styles.content} keyboardDismissMode="on-drag" keyboardShouldPersistTaps="always">
```

Comment:
`keyboardDismissMode="on-drag" keyboardShouldPersistTaps="always"` removed so the keyboard can be dismissed by tapping the screen